dev/AppNotifications/AppNotificationManager.cpp

```diff
@@ -307,7 +329,7 @@ namespace winrt::Microsoft::Windows::AppNotifications::implementation
                     }
                     else
                     { 
-                        registeredClsid = RegisterComActivatorGuidAndAssets();
+                        registeredClsid = RegisterComActivatorGuidAndAssets(winrt::hstring{} /* displayName */, std::wstring{} /* iconUri */);
```

Comment:
This is incorrect too. Don't pass empty arguments please. Instead pass in the overloaded arguments or the one from shortcut/process

Reply:
Actually why do we need to re-register assets at all? Aren't we simply re-routing to another process? @pmpurifoy ? Can you explain this one please and why we need to re-register assets as part of this flow?

Reply:
Call GetActivatorGuid